staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2320,10 +2320,13 @@ type NodeSelector struct {
 	NodeSelectorTerms []NodeSelectorTerm `json:"nodeSelectorTerms" protobuf:"bytes,1,rep,name=nodeSelectorTerms"`
 }
 
-// A null or empty node selector term matches no objects.
+// A null or empty node selector term matches no objects. The requirements of
+// them are ANDed.
 type NodeSelectorTerm struct {
-	//Required. A list of node selector requirements. The requirements are ANDed.
+	// A list of node selector requirements by node's labels.
```

Comment:
Please add `// +optional` to both fields.

Reply:
and omitempty

Reply:
done